dotnet-roslyn/analyzers/Audacia.CodeAnalysis.Analyzers/README.md

```diff
@@ -663,29 +663,4 @@ var filtered = myCollection
 	.Where(a => a.Property != null && a.Property.Value == 1);
 ```
 
-Maximum allowed clauses can be configured in `.editorconfig` by setting `dotnet_diagnostic.ACL1012.max_where_clauses`.
-
-# Custom .editorconfig Settings in Rider
```

Comment:
this hasn't been the case in Rider for a while now, but have been unable to dig out the specific warning.